K3B, KDevelop icons

Ken-Vermette opened the pull request:
K3B, Kdevelop, Amarok icons. Amarok may complain, it's no problem to drop the icon if they do. Note that "K3B, KDevelop icon" is the Amarok icon - it was mis-labeled.

UriHerrera commented:
Feedback:

* Overall - All the icons should have the same height whether they're the same width or not. Ideally, they should not be left in decimals like: 42.098 px but just 42.000 px. I should note that I didn't used radial gradients only linear gradients.
* Amarok - I like the wolf but the green note is overkill, it reminds me of an Oxygen icon. There's also quite the level of detail in the wolf's hair and the wavy shadow is different from the other icons.
* K3B - I like the idea of it but the it's not aligned either and it has also some detail that the icon coudl live without.
* KDevelop - I like the idea here too, the gear isn't aligned though and the light reflection can be removed.

Ken-Vermette commented:
With KDevelop it is actually aligned to the pixel according to the outer-radius of the gear; if you were to fill the gaps between the teeth it would be spot on. I'm going to keep the size/position of the gear as-is, I think it's in our best interest to preserve the diameter over the bounding box, as we have well over a dozen circular icons at that same diameter. I did move the little lip of shade on the bottom from the outside to the inside of the gear, as per your other icons, and the reflection was removed.

K3B; I rejiggered the top-left "from the back" flame to fit within the bounding box of the disc. I also removed the "data lines", and switched the "block letters" of the disc stamp to a line.

Amarok; Yeah, green note was a bad idea - I used it because I had it, more/less. Gone. I played with the hair on the wolf a bit so it's not over-the-top, but still "there" enough to imply some life. I also recall them being more "brand sensitive", so it's a smarter choice to stick closer to the existing design.

Kopete/Cantata/Others: Minor positioning tweaks. Icons which weren't mine using a generic monitor had the "feet" of the monitor adjusted to fit the height requirement.

On a side note I've noticed that Inkscape has been attempting to annoy me with some snapping silliness which has mispositioned some things trying to snap to weird geometry. I've gone through and re-positioned as much as I could spot which was "off-pixel".
